fuzz/README.md

```diff
@@ -0,0 +1,78 @@
+# Fuzzing
+
+Fuzz tests generate a ton of random parameter arguments to the program and then validate that none cause it to crash.
+
+## How does it work?
+
+Typically, Travis CI will run `travis-fuzz.sh` on one of the environments the automated tests are configured for.
+This is the most time-consuming component of the continuous integration workflow, so it is recommended that you detect
+issues locally, and Travis merely acts as a sanity check.
```

Comment:
Its not just because its time-consuming, but fuzzing is only effective with tons and tons of CPU time - if travis found the bug, it was super trivial to hit, but if its in any way subtle, travis doesn't get enough runtime to hit it.

Reply:
Good point.